messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/BridgeAdd.java

```diff
@@ -152,6 +162,10 @@ static BridgeConfiguration createBridgeConfiguration(final OperationContext cont
                 .setUser(user)
                 .setPassword(password);
 
+        if (callTimeout != null) {
```

Comment:
shoudn't be the case since you have a default value

Reply:
Check removed.